cpansa/CPANSA-File-Path.yml

```diff
@@ -149,7 +149,7 @@
     - https://usn.ubuntu.com/94-1/
   reported: 2005-05-02
   severity: ~
-- affected_versions: ">=1.404,<=1.06"
+- affected_versions: "<=1.404,>=1.06"
```

Comment:
This is a change for a different file?

Reply:
I've fixed this up in a separate commit. For some reason I wasn't able to cherry pick this.

Reply:
sorry for duplicate commit

> I've fixed this up in a separate commit. For some reason I wasn't able to cherry pick this.

Reply:
Hey, no problem. I'd rather have a slightly annoying commit than no commit so the problem goes unnoticed. Thanks again.